go/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Move the go directory into $PREFIX/ so it can be built in its install location.
+cd .. && mv go $PREFIX/ && cd $PREFIX/go
```

Comment:
Unfortunately not. It wants to be installed in place. I tried other ways of going about it, but this ended up being the simplest and is what they recommend. See "Installing to a custom location" ( https://golang.org/doc/install ).